src/Build/Collections/RetrievableEntryHashSet/HashHelpers.cs

```diff
@@ -16,8 +17,15 @@ namespace Microsoft.Build.Collections
     /// <summary>
     /// Duplicated because internal to mscorlib
     /// </summary>
-    internal static class HashHelpers
+    internal static partial class HashHelpers
     {
+        public const uint HashCollisionThreshold = 100;
+
+        // This is the maximum prime smaller than Array.MaxArrayLength
+        internal const int MaxPrimeArrayLength = 0x7FEFFFFD;
+
+        public const int HashPrime = 101;
```

Comment:
Where do we use this other than making sure we don't pick an almost-multiple of it?